cmd/node-termination-handler.go

```diff
@@ -196,13 +196,13 @@ func main() {
 
 	for _, fn := range monitoringFns {
 		go func(monitor monitor.Monitor) {
-			log.Info().Str("event_type", monitor.Kind()).Msg("Started monitoring for events")
+			log.Info().Str("monitor_type", monitor.Kind()).Msg("Started monitoring for events")
```

Comment:
Let's use the original field name `event_type` to preserve backwards compatibility, even though that's technically not correct. I'm open to adding another field named `monitor_type` as well, to make the meaning more clear, but we need to keep the existing field name.

Reply:
So you want to keep the `event_type` field but change its value? Old values here were consts like `RebalanceRecommendationKind = "REBALANCE_RECOMMENDATION"` while new ones are consts like `RebalanceRecommendationMonitorKind = "REBALANCE_RECOMMENDATION_MONITOR"`. With that we are keeping the field name but changing its value so we're effectively introducing breaking changes anyway when it comes to people parsing such fields (I guess they are also parsing the field value, not just the mere presence of the field with any value).

Reply:
You're right. I meant that we should keep the original value, too. So `event_type` should be `REBALANCE_RECOMMENDATION`, `SCHEDULED_EVENT`, `SPOT_ITN`, or `SQS_TERMINATE`, rather than the newer `*_MONITOR` values returned by `MonitorKind()`.